roles/openshift_logging/tasks/main.yaml

```diff
@@ -12,6 +12,14 @@
 
 - debug: msg="Created temp dir {{mktemp.stdout}}"
 
+- name: Create local temp directory for doing work in
```

Comment:
@ewolinetz Per our IRC discussion this task moves the local temp directory creation that did not exist off master.  Are we good?  Are there any changes you see that are necessary?